### ISSUE TITLE
Update youtube-dl

### ISSUE THREAD

**Issue by Hummer12007** [tworaz/sailfish-ytplayer#41]
YTPlayer doesn't work for me (until I run the youtube-dl executable with -U key).

**Comment by tworaz** [comment]
0.5.3 already comes with new youtube-dl version. Now it's up to jolla to get it through QA, I can't do anything to speed up this process.

**Comment by Hummer12007** [comment]
Did you consider OpenRepos as a second source? This would speed it up, I think ;)

**Comment by tworaz** [comment]
Nope, but I won't mind if you build the code and upload it to OpenRepos. Right now I don't have any time I could spare to work on YTPlayer.

**Comment by mentaljam** [comment]
With your permission, I will upload a build of YTPlayer to OpenRepos.

**Comment by tworaz** [comment]
Be my guest :)

**Comment by nodefourtytwo** [comment]
I think youtube-dl should be updated again. I can't seem to watch any videos anymore.
